stdlib/os/__init__.pyi

```diff
@@ -346,8 +346,13 @@ class stat_result(structseq[float], tuple[int, int, int, int, int, int, int, flo
     @property
     def st_mtime(self) -> float: ...  # time of most recent content modification,
     # platform dependent (time of most recent metadata change on Unix, or the time of creation on Windows)
-    @property
-    def st_ctime(self) -> float: ...
+    if sys.version_info >= (3, 12) and sys.platform == "win32":
```

Comment:
The replacement `st_birthtime` is only available on 3.12+, so it doesn't make sense to warn on earlier versions.